README.md

```diff
@@ -4,60 +4,150 @@
 [![Discourse topics](https://img.shields.io/discourse/https/discourse.sourcecred.io/topics.svg)](https://discourse.sourcecred.io)
 [![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://sourcecred.io/discord)
 
-SourceCred allows communities to assign Cred scores, which measure the value that contributors have brought to the community.
-The community can then use the scores to distribute rewards (e.g. project-specific Grain tokens) to their contributors.
+SourceCred is an open-source tool that enables online communities to create a
+community-specific contribution score, called Cred, which measures how much
+value every contributor has added to the project. SourceCred then enables the
+project to issue tokens, called Grain, to contributors based on their Cred.
+Grain is purchased by sponsors of the project, and gives sponsors the ability
+to influence Cred scores.
 
-Cred is computed by constructing a Cred Graph showing every contribution to a project, and then running a version of the PageRank
-algorithm to assign scores to every contribution and contributor.
+You can read more at [sourcecred.io].
 
-We currently support loading data from GitHub, Discourse, Discord, and via custom "initiatives".
+[sourcecred.io]: https://sourcecred.io/
 
-Please check out [our website] for more information. If you'd like to get involved as a contributor, please drop by [our Discord]
-and say "hi"!
+## Plugin Architecture
 
-[our website]: https://sourcecred.io/
-[our Discord]: https://sourcecred.io/discord
+SourceCred is organized around a plugin architecture, which ensures that it can
+track and reward any kind of contribution, so long as you can assign addresses
+to contributions, and record how they relate to one another. Currently, we have the following four plugins:
+
+- **GitHub**: Loads GitHub repositories, including issues, pull requests, and reviews
+- **Discourse**: Loads Discourse forums, including posts, topics, and likes
+- **Discord**: Loads Discord servers, including messages and reactions
+- **Initiatives**: Loads manually added contributions. Still in alpha.
+
+Every plugin has a two-part name in the form `$OWNER/$NAME`; for example,
+SourceCred's own GitHub plugin is named `sourcecred/github`.
```

Comment:
Should specify the name of each plugin directly so people know what to put when calling `sourcecred load`:

```suggestion
- **sourcecred/github**: Loads GitHub repositories, including issues, pull requests, and reviews
- **sourcecred/discourse**: Loads Discourse forums, including posts, topics, and likes
- **sourcecred/discord**: Loads Discord servers, including messages and reactions
- **sourcecred/initiatives**: Loads manually added contributions. Still in alpha.

Every plugin has a two-part name in the form `$OWNER/$NAME`; for example,
SourceCred's own GitHub plugin is named `sourcecred/github`.
```

Reply:
Also should include the environment variables the user needs to setup for each plugin, and a link to instructions on how they can get those tokens. Mostly for Discord and Github. Here's a good guide: https://www.writebots.com/discord-bot-token/#:~:text=A%20Discord%20Bot%20Token%20is,generate%20a%20Discord%20Bot%20Token.